plugin/sublime-text/.pylintrc

```diff
@@ -0,0 +1,436 @@
+# This Pylint rcfile contains a best-effort configuration to uphold the
```

Comment:
There are some error messages that still need to be turned off, so instead of using pylint's default settings completely, I decided to simplify the current settings: 5e286af1ac3e1acad6b90d563ebe79737a08baa8